manifests/profiles/openshift.yaml

```diff
@@ -5,6 +5,13 @@ spec:
     cni:
       enabled: true
       namespace: kube-system
+      k8s:
+        overlays:
+          - kind: DaemonSet
+            name: istio-cni-node
+            patches:
+              - path: spec.template.spec.hostNetwork
```

Comment:
```suggestion
              - path: spec.template.spec.containers[0].securityContext.privileged
```

Reply:
For a reason I don't know yet (will probably know more tomorrow), setting `hostNetwork` on OpenShift implies in setting `privileged` for the container. We don't really need `hostNetwork`, so setting `privileged` is enough.